RestaurantManagementSystem/RestaurantManagementSystem/src/Core/RestaurantRequest/Entities/RegistrationRequest.cs

```diff
@@ -58,21 +61,50 @@ public class RegistrationRequest
             return new RegistrationRequest(id, name, email, address, Contact, verificationcode, Status, DateCreated, VerificationCodeExpiresAt);
         }
 
+        private List<INotification> _domainEvents= new List<INotification>();
+        public List<INotification> DomainEvents => _domainEvents;
```

Comment:
does changing on this `DomainEvents` will change the internal `_domainEvents` is there a way to make it readOnly field?